mp/src/game/server/neo/neo_player.cpp

```diff
@@ -2274,7 +2275,7 @@ bool CNEO_Player::ProcessTeamSwitchRequest(int iTeam)
 		State_Transition(STATE_OBSERVER_MODE);
 	}
 	else if (iTeam == TEAM_JINRAI || iTeam == TEAM_NSF)
-	{
+	{		
```

Comment:
Since this is going through another round anyway, I'm reopening this as it's unaddressed - if you think it's not worth fixing then feel free to re-close.